pkg/util/util.go

```diff
@@ -0,0 +1,37 @@
+package util
```

Comment:
Should this be a part of the public API at all? On the one hand it clearly can be useful elsewhere, OTOH it requires extra care to design it to last.

If this should be public, please make it a separate subpackage, `pkg/iolimits` (??) or something like that; an unspecific `pkg/util` that grows over time does not _quite_ hurt but it will drag in extra code and dependencies for every caller that only wants one of the utilities.

Reply:
I think it should be `./internal` see https://github.com/containers/image/pull/803#discussion_r372444310

Reply:
Now in `./internal/iolimits` :+1: